google_analytics/ga_homepage_report.py

```diff
@@ -444,7 +454,8 @@ def output_report(filename, total_homepage_views=None, total_course_card_clicks=
         homepage_subject_worksheet.set_column('C:S', 15, percent_fmt)
 
         # Write heading
-        homepage_subject_worksheet.write('A1', 'Top Subject Pages from the Homepage, Data from '+str(start_date)+' to '+str(end_date) , cell_format)
+        homepage_subject_worksheet.write('A1', 'Top Subject Pages from the Homepage, Data from ' + str(
```

Comment:
Alternatively, we can construct the string in a named variable before passing it to this method.
```
# Note: a different variable name might be more appropriate, as I'm not familiar with this method call.
heading = 'Top Subject Pages from the Homepage, Data from ' + str(start_date) + ' to ' + str(end_date)
homepage_subject_worksheet.write('A1', heading, cell_format)
```